### ISSUE TITLE
Chiari Type II malformation

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7248]
**Preferred term label:** Chiari Type II malformation

**Synonyms** Arnold-Chiari type II malformation, Chiari II malformation

**Definition (free text, please give PubMed ID)**
 While Chiari I malformation is thought to result from a small posterior fossa, Chiari II occurs due to in utero malformation of the spine and cranial structures.  The underlying cause of Chiari II malformation is thought to be an in utero CSF leak due to open spinal dysraphism, resulting in a small posterior fossa with distortion of the cerebellum and a diminutive cisterna magna, sometimes accompanied by ventriculomegaly and an altered shape of the calvarium.  _Chiari II malformation is always associated with spinal myelomeningocele or myelocele._ PMID:32706613)
**Comments** Classical signs described on prenatal ultrasound include : lemon sign and 
banana cerebellum sign 
**Parent term (use hpo.jax.org/app)**
Arnold-Chiari malformation HP:0002308
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Thanks, @patt1morgan  I added this information.
